bigquery/cloud-client/simple_app.py

```diff
@@ -15,15 +15,42 @@
 # limitations under the License.
 
 """Simple application that performs a query with BigQuery."""
+
 # [START bigquery_simple_app_all]
+import argparse
+
 # [START bigquery_simple_app_deps]
+import google_auth_oauthlib
 from google.cloud import bigquery
 # [END bigquery_simple_app_deps]
 
 
-def query_stackoverflow():
+def authenticate(client_id, client_secret):
+    # [START bigquery_simple_app_authenticate]
+    # TODO(developer): Set the client ID and client secret to the values
```

Comment:
This TODO requires a lot of background understanding.  Should we link to more detailed documentation about setting up/managing client ID and secret?  This could also be the case where we supply a default client ID for the samples, with commentary about replacing them for your own application to avoid blocking someone on this extra level of setup.